ros2controlcli/ros2controlcli/verb/list_controllers.py

```diff
@@ -21,6 +21,31 @@
 from ros2controlcli.api import add_controller_mgr_parsers
 
 
+def print_controller_state(c, args):
```

Comment:
Could you post a figure of the output? So, we can optimize it immediately if needed. We can make it more colorful and better aligned to be more readable. You are free to make a proposal too! :)

If you think that this would be better to do in a follow-up PR, we should open an issue.

Reply:
I think a follow-on PR would be better mainly because I am just following the existing pattern in this one. I think this [one](https://github.com/ros-controls/ros2_control/pull/763) would be a good place to make the change you suggest as well.

Reply:
Changes will be done in #763 then.